Monika After Story/game/zz_music_selector.rpy

```diff
@@ -26,6 +27,7 @@ init -1 python in songs:
         "<loop 17.451 to 119.999>mod_assets/bgm/ddlc_maintheme_80s.ogg"
     )
     FP_SAYO_NARA = "<loop 36.782>bgm/d.ogg"
+    FP_PLAYWITHME_VAR6 = "bgm/6s.ogg"
```

Comment:
Mmm I'll listen to this later, but I'm almost 100% certain that you're gonna need to make this loop seamlessly. Can you add a loop tag? The other songs have it, you should be able to infer how it works.

Reply:
I'll work on it.